### DISCUSSION TITLE
PyInvest

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#837] [優秀Python套件]
PyInvest是一個Python量化分析和金融投資的Blog
也有Python模組、機器學習的教學影片
是一個很好的學習資源

網站
https://pyecontech.com/
Youtube頻道
https://www.youtube.com/c/Pyinvest

**Comment by Pietra1226** [comment]
謝謝分享，雖然我不太懂金融，但這個網站確實很多實用的知識

**Comment by williamyen0623** [comment]
感謝分享，python在金融市場上的應用相當多元，這是很好的學習資源

**Comment by Kotori0501** [comment]
感謝分想，雖然我也不太懂金融相關領域，但我覺得學程式可以多看看在不同領域間的應用並了解，說不定在未來也會使用到

**Comment by dabbie109303501** [comment]
python讓你...(x)學會寫程式 (o)學到金融知識

**Comment by ruthchen1020** [comment]
之前做過相關主題的報告，覺得影片很實用也很值得分享。

**Comment by StevenWanglolz** [comment]
沒想到Python也可以應用在金融投資上，難怪老師這麼強調Python的跨領域應用!

**Comment by Evan0979** [comment]
感覺可以應用在財金領域上,感謝分享!!

**Comment by Doreen331** [comment]
之前在研究投資的時候就有看到有大神利用程式幫他盯盤，那時候覺得也太方便了吧，果然厲害的人會有錢不是沒有原因的，但不是我🥲
